crates/ra_batch/src/lib.rs

```diff
@@ -126,3 +126,27 @@ impl BatchDatabase {
         Ok((db, local_roots))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use ra_hir::Crate;
+    use super::*;
+
+    #[test]
+    fn test_loading_rust_analyzer() {
+        let mut path = std::env::current_exe().unwrap();
+        while !path.join("Cargo.toml").is_file() {
```

Comment:
In the future, we should probably categorize tests that touch cargo or do other IO and skip them by default.... Also it would be cool to somehow schedule such tests to run first and in parallel to CPU-bound tests. Unfortunately Cargo can’t parallelise across test binaries

Reply:
Is that possible with current cargo? I.e. I think you can only explicitly skip tests, not have tests that are skipped by default and only run when explicitly specified?

Reply:
Yeah, there are no good solutions here. We can do if not env(“CI”) or something like that, but I am worrying about CI times as well, and I don’t know even hacky solutions for the scheduling problem